### PR TITLE
Update Localization instructions

### PR DESCRIPTION
**Pull request by Siedlerchr**
Corrected gradlew task name
#1312 
@koppor  @stefan-kolb

### CODE REVIEW

**Comment by simonharrer** [comment]
Please check the order of the imports: https://travis-ci.org/JabRef/jabref/builds/126478088

**Comment by Siedlerchr** [comment]
Hm, will check my Eclipse settings again,.

**Comment by koppor** [comment]
`gradlew eclipse` should be enough --> We are generating the correct settings. No need for manual intervention.

File: src/test/java/net/sf/jabref/logic/l10n/LocalizationConsistencyTest.java [hunk 1]
```diff
@@ -141,8 +141,7 @@ public void findObsoleteLocalizationKeys() throws IOException {
             System.out.println(obsoleteKeys.stream().map(Object::toString).collect(Collectors.joining("\n")));
             System.out.println();
             System.out.println("1. REMOVE THESE FROM THE ENGLISH LANGUAGE FILE");
-            System.out.println(
-                    "2. EXECUTE gradlew -b localization.gradle compareAndUpdateTranslationsWithEnglishTranslation TO");
+			System.out.println("2. EXECUTE gradlew -b localization.gradle generateMissingTranslationKeys TO");
```
Comment (koppor) [inline src/test/java/net/sf/jabref/logic/l10n/LocalizationConsistencyTest.java]: Wrong indent?

Comment (Siedlerchr) [inline src/test/java/net/sf/jabref/logic/l10n/LocalizationConsistencyTest.java]: Hm. Will look into that. And I use gradlew Eclipse.

**Comment by Siedlerchr** [comment]
@koppor  Fixed the import order. The setting was not activated in my Eclipse saving options.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -96,9 +96,14 @@ because <additional rationale>.
 
 
 ### When adding a new Localization.lang entry
-Run `python scripts/syncLang.py -s` to search for new untranslated strings.
-If everything is OK, run `python scripts/syncLang.py -s -u` to update `JabRef_en.properties`.
-
+Add new Localization.lang("KEY") to Java file.
+Tests fail. In the test output a snippet is generated which must be added to the English translation file. There is also a snippet generated for the non-English files, but this is irrelevant.
```
Comment (tobiasdiez) [inline CONTRIBUTING.md]: @JabRef/developers Can these irrelevant statements for the other languages be removed?

**Comment by tobiasdiez** [comment]
LGTM :+1:

**Pull request merged**